jmp-cloud-service-impl/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>Module_3</artifactId>
+		<groupId>org.example</groupId>
+		<version>1.0-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>jmp-cloud-service-impl</artifactId>
+	<dependencies>
+		<dependency>
+			<groupId>org.example</groupId>
+			<artifactId>jmp_dto</artifactId>
+			<version>1.0-SNAPSHOT</version>
+			<scope>compile</scope>
```

Comment:
Compile is default scope. You don't have to specify it explicitly.
Optional: delete scope tag.